cowprotocol/cowswap
# Fix cosmos locally

# Summary

I couldn't get Cosmos to run locally

With Anxo's help, it did after a `yarn upgrade` on a fresh clone.

Did it using latest v16 LTS `v16.18.1`

  # To Test

1. Locally, run `yarn` then `yarn cosmos:run`
2. Open localhost:5001
* Cosmos should work as usual
3. On CI, open [pr link]/cosmos
* Cosmos should work as usual
